Source/DafnyCore/AST/DafnyAst.cs

```diff
@@ -27,6 +27,7 @@ public class FilledInDuringResolutionAttribute : System.Attribute { }
   public abstract class INode {
 
     public IToken tok = Token.NoToken;
+    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
```

Comment:
Why do we need this attribute?

Reply:
Good question, indeed it's not relevant to the problem to be solved directly.
I had read when learning about this attribute that it can be used to avoid displaying duplicate field and wrapping properties (such as tok and Tok) in the debugger. I think it is clever, so I'm enabling it. One might counter-argue that they will look for "Tok" and not think of looking at "tok", if that ever happens, we can revert that. What do you think?

Reply:
Can't we make `tok` private? Then it doesn't show up in the standard list.

Reply:
I don't know how you configured your debugger, but private fields do show up my debugger. I would be very annoyed if they weren't, after all, what's the point of debugging if you can't figure out the internal state of the object?

Reply:
In my debugger they're hidden under a separate menu, so they don't create noise unless you explicitly look for them.